assets/prometheus-k8s/cluster-role.yaml

```diff
@@ -8,7 +8,6 @@ metadata:
     app.kubernetes.io/part-of: openshift-monitoring
     app.kubernetes.io/version: 2.32.1
   name: prometheus-k8s
-  namespace: openshift-monitoring
```

Comment:
Why these namespace are removed?

Reply:
I didn't remove it explicitly like I mentioned in description after running generate it did a lot of changes unrelated to this bug. I am checking why it happend

Reply:
Ok looks like its due to https://github.com/prometheus-operator/kube-prometheus/pull/1566 we should override this in config to fix

Reply:
But why do we need a `namespace` field for clusterrole and clusterrole binding since its not a namespaced resource

Reply:
Ahh, thats correct, We don't need a namespace for cluster role